AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="cell_info_title">Neighboring Cells</string>
```

Comment:
@larsgrefer, thanks for clarifying, makes sense. But do you think why did you change [this](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/NeighboringConsistency/AIMSICD/src/main/res/values/translatable_strings.xml#L321) again?